messaging/src/main/java/ids/messaging/protocol/http/IdsHttpService.java

```diff
@@ -416,16 +437,16 @@ private Response sendRequest(final Request request,
     }
 
     /**
-     * Logs the request message to be send in debug-level log.
+     * Logs the request message to be send in log.
      *
      * @param request The request message to be send.
      * @throws IOException If Buffer could not be accessed.
      */
-    private void debugLogRequest(final Request request) throws IOException {
+    private void logRequest(final Request request) throws IOException {
```

Comment:
Why is this method extracted when it just has one place calling this method? At line 418 the response logging is handled inside the method too.

Reply:
This is done to reduce the `Cognitive Complexity` as part of code-quality measurements of methods, which should not be too high, else goes as `Code smell`. 
`Cognitive Complexity is a measure of how hard the control flow of a method is to understand. Methods with high Cognitive Complexity will be difficult to maintain.`